src/components/toast/toast.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { Toast as ToastData, toast, resolveValue } from 'react-hot-toast';
+import { CloseIcon } from '~/src/components/icons';
+
+import styles from './toast.module.scss';
+
+export { toast, type ToastData };
+
+const dismissToast = (id: string) => toast.dismiss(id);
+
+export const Toast = ({ toast }: { toast: ToastData }) => {
+    const variantClassName = {
+        success: styles.success,
+        error: styles.error,
+        blank: styles.blank,
+        loading: styles.loading,
+        custom: '',
+    }[toast.type];
+
+    const className = classNames(
```

Comment:
If we move classNames into variable, we loose className in ElementTree.
Subjectively, I use to see a className there, so having just root `div` scares me a bit (because I know there should be a class).

<img width="905" alt="Screenshot 2024-11-08 at 15 02 34" src="https://github.com/user-attachments/assets/739b8201-c289-44d2-8265-1a85f8b6413a">

Reply:
The code looks worse imo, but OK.